validators/terravaloper1hz754zdldnrrhp3qpfan3l2dxkcv5cgkuzqq9v/template/profile.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Please put this file in

`validators/terravaloper1hz754zdldnrrhp3qpfan3l2dxkcv5cgkuzqq9v/profile.json`

Thanks